_layouts/default.html

```diff
@@ -50,12 +63,11 @@
 				var s = document.getElementsByTagName('script')[0];
 				s.parentNode.insertBefore(o, s);
 			}
-		}('https://www.ft.com/__origami/service/build/v2/bundles/js?modules=o-normalise@^1.0.0,o-typography@^5.8.0,o-layout@^3.0.0,o-header-services@^3.0.0,o-footer-services@^2.0.3,o-syntax-highlight@^1.4.0,o-message@^3.1.0,o-table@^7.2.1'));
+		}('https://www.ft.com/__origami/service/build/v2/bundles/js?modules=o-normalise@^2.0.0,o-typography@^6.0.0,o-layout@^4.0.0,o-header-services@^4.0.0,o-footer-services@^3.0.0,o-syntax-highlight@^3.0.0,o-message@^4.0.0,o-table@^8.0.0'));
```

Comment:
Use latest o-layout for sure
```suggestion
		}('https://www.ft.com/__origami/service/build/v2/bundles/js?modules=o-normalise@^2.0.0,o-typography@^6.0.0,o-layout@^4.0.4,o-header-services@^4.0.0,o-footer-services@^3.0.0,o-syntax-highlight@^3.0.0,o-message@^4.0.0,o-table@^8.0.0'));
```